portability-worker/src/main/resources/settings/common.yaml

```diff
@@ -1,2 +1,10 @@
 env: LOCAL
-cloud: LOCAL
\ No newline at end of file
+cloud: LOCAL
+serviceProviderClasses: [
```

Comment:
Isn't this generated by the build script?

Reply:
It is removed.